### PR TITLE
Cisco wsa

### PR DESCRIPTION
**Pull request by Nader-Elgohary**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Allow users to execute commands against CiscoWSA.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 
2021-11-06 07_37_06-root@W-CND1223G4B_ ~_demisto_content

### CODE REVIEW

**Comment by Nader-Elgohary** [comment]
Hi @iyeshaya,
Most probably I managed and solved the issues from my side, do you think there is something else to be done?

**Comment by Nader-Elgohary** [comment]
@iyeshaya 
Regarding the F5LTM.yml error which is under "ci/circleci: Run Validations" section, this was done by mistake, and actually, I don`t know what should I do to solve it?

But regarding CiscoWSA, I managed to solve the old errors by committing the last version (1.v18).

Looking forward to hear from you soonest.

**Comment by Nader-Elgohary** [comment]
Hi @iyeshaya,

Please your feedback on the next step if required from my side, otherwise, What should we do to move forward?

Thanks in advance.

File: Packs/F5LTM/Integrations/F5LTM/F5LTM.yml [hunk 1]
```diff
@@ -563,4 +563,4 @@ script:
   type: python
 tests:
 - No tests (auto formatted)
-fromversion: 6.0.0
+fromversion: 6.0.0
```
Comment (iyeshaya) [inline Packs/F5LTM/Integrations/F5LTM/F5LTM.yml:566]:
```suggestion
fromversion: 6.0.0

```

**Review by iyeshaya — APPROVED** [review]

**Pull request merged**